### PR TITLE
Temporarily skip qos and pfcwd test scripts in PR testing

### PR DESCRIPTION
**Pull request by yutongzhang-microsoft**
### Description of PR

Since the test scripts for `qos` and `pfcwd` are not yet included in PR testing, we have temporarily applied conditional marks to skip them. Once these scripts are integrated into PR testing, this PR can be reverted.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
Since the test scripts for `qos` and `pfcwd` are not yet included in PR testing, we have temporarily applied conditional marks to skip them. Once these scripts are integrated into PR testing, this PR can be reverted.

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -1392,6 +1400,7 @@ qos/test_buffer.py:
     conditions:
       - "asic_type in ['cisco-8000'] or 't2' in topo_name"
       - "topo_type in ['m0', 'mx']"
+      - "asic_type in ['vs']"
```
Comment (xwjiang-ms) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1403]: qos/test_buffer.py is already in PR test and running good

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 2]
```diff
@@ -1334,6 +1336,7 @@ pfcwd/test_pfc_config.py::TestPfcConfig::test_forward_action_cfg:
      conditions:
         - "asic_type in ['cisco-8000']"
         - "topo_type in ['m0', 'mx']"
+        - "asic_type in ['vs']"
```
Comment (xwjiang-ms) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1339]: pfcwd/test_pfc_config.py is included in PR test and running good, no need to skip

**Review by xwjiang-ms — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Comment by wangxin** [comment]
@XuChen-MSFT  Can you help review this change?

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 3]
```diff
@@ -1468,6 +1517,7 @@ qos/test_qos_sai.py::TestQosSai::testIPIPQosSaiDscpToPgMapping:
       - https://github.com/sonic-net/sonic-mgmt/issues/12906
       - "topo_type in ['m0', 'mx']"
       - "topo_name not in ['t0', 't0-64', 't0-116', 't0-35', 't0-56', 't0-standalone-32', 't0-standalone-64', 't0-standalone-128', 't0-standalone-256', 'dualtor-56', 'dualtor-120', 'dualtor', 't0-80', 't0-backend', 't1-lag', 't1-64-lag', 't1-56-lag', 't1-backend', 't2', 't2_2lc_36p-masic', 't2_2lc_min_ports-masic'] and asic_type not in ['mellanox']"
+      - "asic_type in ['vs']"
```
Comment (XuChen-MSFT) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1520]: Can skip test for "qos/test_qos_sai.py" instead of each qos case?

Comment (yutongzhang-microsoft) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1520]: As our principle of conditional mark, if we have longer matching, we will only adopt the conditions in the longest matching. This means, if we have both `qos/test_qos_sai.py` and  `qos/test_qos_sai.py::<test case>`, we will use the conditions in the second entry.

**Review by XuChen-MSFT — APPROVED** [review]
All good!

**Pull request merged**